lib/ex_unit/test/ex_unit/formatter_test.exs

```diff
@@ -469,6 +469,58 @@ defmodule ExUnit.FormatterTest do
            """
   end
 
+  test "formats nested maps with column limit" do
```

Comment:
Could we have a test in `diff_test.exs` instead?

Reply:
Good should, updated